sap-spring-properties/src/main/java/com/sap/cloud/environment/servicebinding/environment/SapServiceBindingEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,34 @@
+package com.sap.cloud.environment.servicebinding.environment;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.context.properties.bind.BindResult;
+import org.springframework.boot.context.properties.bind.Bindable;
+import org.springframework.boot.context.properties.bind.Binder;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.annotation.Order;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+import com.sap.cloud.environment.servicebinding.SapServiceBindingsProperties;
+
+/* An {@link EnvironmentPostProcessor} that binds service binding properties to {@link SapServiceBindingsProperties}. */
+@Order
+public class SapServiceBindingEnvironmentPostProcessor implements EnvironmentPostProcessor
+{
+    private static final Logger log = LoggerFactory.getLogger(SapServiceBindingEnvironmentPostProcessor.class);
+
+    @Override
+    public void postProcessEnvironment( ConfigurableEnvironment environment, SpringApplication application )
+    {
+        Bindable<SapServiceBindingsProperties> bindable = Bindable.of(SapServiceBindingsProperties.class);
+
+        BindResult<?> bindResult = Binder.get(environment).bind("services", bindable);
+        if( bindResult.isBound() ) {
+            SapServiceBindingsPropertiesAccessor
+                .setServiceBindingsProperties(((SapServiceBindingsProperties) bindResult.get()).getServiceBindings());
+        } else {
+            log.info("Could not bind any service from the properties.");
```

Comment:
```suggestion
            log.debug("Could not bind any service from the properties.");
```

This might happen if no `services` section is defined. I think we shouldn't log that information on every startup. Rather log positive things on `info` than negative things. If you'd like a log message on info rather move it to the case that properties are bound and inform about that.